tools/pipelines/templates/include-test-coverage.yml

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-test-coverage
+
+parameters:
```

Comment:
This looks very much like include-test-real-service.yml.   What is the difference? Can we reuse?

Reply:
We are going to use the `@fluid-internal/local-server-tests test package` to detect flaky tests. Reusing` include-test-real-service.yml` is doable from the view of syntax, but we do not use test-end-to-end package here, it is not related to the realsvc. It might be better to rename that template yaml file to cover both cases if we need to reuse.